<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 228, 255);</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>471</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_5">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 228, 255);</string>
      </property>
      <property name="text">
       <string>ID. Avion</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 228, 255);</string>
      </property>
      <property name="text">
       <string>Modelo</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_17">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 228, 255);</string>
      </property>
      <property name="text">
       <string>Peso Nominal</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_18">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 228, 255);</string>
      </property>
      <property name="text">
       <string>Capacidad</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_15">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 228, 255);</string>
      </property>
      <property name="text">
       <string>Tipo propulsion</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_16">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 228, 255);</string>
      </property>
      <property name="text">
       <string>N° Motor</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Reacción</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Turbo Hélice</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hélice</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/cct/iconoregistroavion.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_registrar_piloto_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="logo usuario.qrc"/>
 </resources>
 <connections/>
</ui>
